vue-app/src/views/VerifyLanding.vue

```diff
@@ -74,10 +71,6 @@ import WalletWidget from '@/components/WalletWidget.vue'
   components: { ProgressBar, RoundStatusBanner, WalletWidget },
 })
 export default class VerifyLanding extends Vue {
-  get walletProvider(): any {
```

Comment:
Why don't we want to check for this anymore?

Reply:
Ah, because we have WalletConnect now?

Reply:
Yes, the thing is that before we where doing this:
```js
get walletProvider() {
  return window.ethereum
}
```
and you could certainly have a provider and no user. But now, you can't. You will always going to have both or neither.

Reply:
When I did the wallet connect PR I refactored this to read it from `this.$store.state.currentUser?.walletProvider`.